unlock_cond_state_ctrl.go

```diff
@@ -19,6 +19,11 @@ func (s *StateControllerAddressUnlockCondition) VBytes(rentStruct *RentStructure
 		s.Address.VBytes(rentStruct, nil)
 }
 
+func (s *StateControllerAddressUnlockCondition) WorkScore(workScoreStructure *WorkScoreStructure) WorkScore {
+	// StateControllerAddressUnlockCondition requires a signature check, but on consumption, not creation.
+	return workScoreStructure.FactorData.Multiply(s.Size())
```

Comment:
Or we switch to a "creator pays principle" and charge everyone that creates an output that has these features.
So we would charge him for the signature check here as well.

Reply:
I think we now agreed we will go with consumer pays and charge for signatures through the unlocks.